rosco-web/config/packer/aws-ebs.json

```diff
@@ -38,7 +38,8 @@
     "type": "shell",
     "script": "{{user `configDir`}}/install_packages.sh",
     "environment_vars": [
-      "deb_repo={{user `deb_repo`}}",
+      "repository={{user `repository`}}",
+      "package_type={{user `package_type`}}",
```

Comment:
Please ensure that you keep the aws and gce templates consistent where possible. As this PR stands now, assuming the missing variables were defined, the gce template would cease to function properly as it would not be possible receive the (optional) `deb_repo`. It would also be passing that to the env of the `install_packages.sh` which no longer respects it.